### DISCUSSION TITLE
Is GNU coreutils a project goal?

### DISCUSSION THREAD

**Discussion by jaskij** [uutils/coreutils#4369] [General]
Reading the README, one of the points is [Improving GNU compatibility](https://github.com/uutils/coreutils#improving-the-gnu-compatibility). Is GNU coreutils compatibility an explicit project goal, or only a nice to have?

**Comment by tertsdiepraam** [comment]
Yes, uutils is specifically a reimplementation of the GNU coreutils, including GNU extensions.
